### PR TITLE
Removed tl;dr

### PR DESCRIPTION
**Pull request by Vikkybliz**
Removed tl;dr.

tl:dr isn't necessary 

Some see it as a rude acronym
*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by goderbauer** [comment]
Change looks good, but you will have to sign the CLA for us to accept it.

**Comment by Vikkybliz** [comment]
Signed.

**Comment by Hixie** [comment]
The reason I put "tl;dr" there was so that people could distinguish the "abstract" part (summary of the whole page) from the actual content. I'm not sure just removing it is a good idea, since it doesn't indicate that this part is relevant in that way.

We could have "Executive summary:", but I'm not sure that's any less rude than "tl;dr"? I'm not sure why "tl;dr" is considered rude...
